pkgs/development/python-modules/sphinx/default.nix

```diff
@@ -30,38 +30,39 @@
 
 # check phase
 , cython
+, filelock
 , html5lib
 , pytestCheckHook
-, typed-ast
 }:
-
+# Change ä to æ in file names, since ä can be encoded multiple ways on different
+# filesystems, leading to different hashes on different platforms.
+let normalizeFileNames = ''
+  cd "$out";
+  mv tests/roots/test-images/{testimäge,testimæge}.png
+  sed -i 's/testimäge/testimæge/g' tests/{test_build*.py,roots/test-images/index.rst}
+'';
+in
 buildPythonPackage rec {
   pname = "sphinx";
-  version = "5.3.0";
+  version = "7.0.1";
   format = "pyproject";
-
-  disabled = pythonOlder "3.6";
+  disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = "sphinx-doc";
     repo = pname;
     rev = "refs/tags/v${version}";
-    hash = "sha256-80bVg1rfBebgSOKbWkzP84vpm39iLgM8lWlVD64nSsQ=";
-    postFetch = ''
-      cd $out
-      mv tests/roots/test-images/testimäge.png \
-        tests/roots/test-images/testimæge.png
-      patch -p1 < ${./0001-test-images-Use-normalization-equivalent-character.patch}
-    '';
+    postFetch = normalizeFileNames;
```

Comment:
```suggestion
    postFetch = ''
      # Change ä to æ in file names, since ä can be encoded multiple ways on different
     # filesystems, leading to different hashes on different platforms.
     cd "$out";
     mv tests/roots/test-images/{testimäge,testimæge}.png
     sed -i 's/testimäge/testimæge/g' tests/{test_build*.py,roots/test-images/index.rst}
  '';
```